gpt_engineer/steps.py

```diff
@@ -15,7 +15,8 @@
 
 def setup_sys_prompt(dbs: DBs) -> str:
     return (
-        dbs.preprompts["generate"] + "\nUseful to know:\n" + dbs.preprompts["philosophy"]
+        dbs.preprompts["generate"] + "\nUseful to know:\n" +
+        dbs.preprompts["philosophy"]
```

Comment:
We're using line width = 90. It's set in pyproject.toml.

Do you think you could let your black respect this config? (otherwise the CI will fail on main, we don't want that)